src/components/Navigation/Navigation.jsx

```diff
@@ -119,9 +119,19 @@ export default class Navigation extends React.Component {
       });
 
       window.addEventListener('keyup', e => {
-        if (e.which === 9 && e.target.classList.contains('navigation__search-input')) {
-          this._openSearch();
+
+        switch(e.which) {
```

Comment:
I think it makes sense to check for `ctrlKey`, `shiftKey` etc status and if they are pressed don't even go into the switch eg. for example for `CMD + ESC` or any other reasons like `shift + /`

Reply:
```suggestion
        if (e.shiftKey || e.ctrlKey || e.metaKey) return;
        switch(e.which) {
```

Reply:
Thanks mate, all done :)